apps/web/src/app/_components/footer.tsx

```diff
@@ -18,9 +19,17 @@ export default function Footer() {
         />
         <h2 className="text-xl font-semibold">Checkout cooper on GitHub</h2>
       </Link>
-      <a href="https://clearbit.com" target="_blank" className="italic">
-        Logos provided by Clearbit
-      </a>
+      <div className="flex flex-col items-center md:flex-row md:gap-6">
```

Comment:
I say we add `gap-2` here so that when they are vertically aligned they aren't as close together